src/JoomlaBrowser.php

```diff
@@ -897,4 +897,36 @@ public function disableStatistics()
         $I->waitForElement(['link' => 'Never'], 60);
         $I->click(['link' => 'Never']);
     }
+    
+    /**
+     * Function to delete the menu item
+     *
+     * @param  string  $menuItem   Menu Item to be deleted
+     * @param  string  $menu    Menu type of the menu item
+     *
+     * @return void
+     */
+    public function deleteMenuItem($menuItem,$menu)
+    {
+        $I->amOnPage('administrator/index.php?option=com_menus&view=menus');
+        $click(['xpath' => "//a[contains(text(),".$menu.")]"]);
+        $I->waitForText('Menus: items ('.$menu.')', '30', ['css' => 'h1']);
+        $I->searchForItem($title);
+        $I->click(['xpath' => "//input[@id='cb0']"]);
+        $I->click(['xpath'=> "//button[@onclick=\"if (document.adminForm.boxchecked.value==0){alert('Please first make a selection from the list.');}else{ Joomla.submitbutton('items.trash')}\"]"]);
+        $I->waitForText('Menus: Items (Main Menu)','30',['css' => 'h1']);
+        $I->expectTo('see a success message and the menu removed from the list');
+        $I->waitForText('1 menu item successfully trashed.',['id' => 'system-message-container']);
+        $I->see('1 menu item successfully trashed.',['id' => 'system-message-container']);
+        $I->searchForItem($title);
+        $I->waitForText('No Matching Results',60, ['class' => 'alert-no-items']);
+        $I->see('There are no menu items matching your query.', ['class' => 'alert-no-items']);
+
+        //search the menuitem from the search bar and then trash it.
```

Comment:
If they make sense (don't explain obvious things). But there is also a code style for that, after the two slashes //, make one space and the first letter has to be uppercase.

```
// Correct
//not
```

Which IDE are you using? There is a github repository for the joomla code style (including configuration files for Netbeans, Eclipse and PhpStorm) and you can use phpcs with Joomla sniffer rules to validate it on the fly. 

https://github.com/joomla/coding-standards